TLM/TLM/Manager/Impl/SpeedLimitManager.cs

```diff
@@ -240,7 +240,7 @@ public class SpeedLimitManager
         public float GetGameSpeedLimit(uint laneId) {
             return GetGameSpeedLimit(
                 laneId: laneId,
-                laneInfo: LaneUtil.GetLaneInfo(laneId));
+                laneInfo: ExtLaneManager.Instance.GetLaneInfo(laneId));
```

Comment:
I'm already wishing there was an extension:

```csharp
laneId.GetLaneInfo()
```

Reply:
I don't like the idea of hiding the fact that the manager is involved. Managers might do a lot of things that you wouldn't expect when calling a "get the object for this ID" kind of method.